backend/src/utils/notebookUtils.ts

```diff
@@ -438,7 +438,7 @@ export const createNotebook = async (
     notebookAssembled.metadata.annotations = {};
   }
 
-  notebookAssembled.metadata.annotations['notebooks.opendatahub.io/inject-oauth'] = 'true';
+  notebookAssembled.metadata.annotations['notebooks.opendatahub.io/inject-oauth'] = 'false';
```

Comment:
I wonder - can we just not set it at all instead?

Reply:
I'll test this in another branch and update accordingly

Reply:
Yes, tested and simply not touching the inject annotation works.